src/http_rest_api_handler.cpp

```diff
@@ -92,6 +103,9 @@ const std::string HttpRestApiHandler::kfs_serverliveRegexExp =
 const std::string HttpRestApiHandler::kfs_servermetadataRegexExp =
     R"(/v2)";
 
+const std::string HttpRestApiHandler::oai_chatCompletionRegexExp =
```

Comment:
```suggestion
const std::string HttpRestApiHandler::openai_chatCompletionRegexExp =
```

Reply:
And what do you think about enum class member being named OAI_ChatCompletion? It is aligned with KFS_ TFS_